### PR TITLE
docs: add more documentation about lint messages

### PR DESCRIPTION
**Pull request by bob-watson**
Adds suggestions for how to resolve the most common messages produced by the documentation linter.

Adds links to the Proselint and other style tests to the new documentation.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Some documentation style rule tests don't have links to more information about the style rule.

Issue Number: N/A

## What is the new behavior?
Adds a new page that explains more about the most common documentation linter problems to the AIO docs.
Adds links to the documentation style rule tests that didn't have FMI links for their style rule.
NOTE: some style rules still don't have help or help links, yet.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by dgp1130 — APPROVED** [review]

**Comment by bob-watson** [comment]
The new doc created by this PR can be viewed in its rendered form at: * https://pr47019-757d0c8.ngbuilds.io/guide/docs-lint-errors
The doc topic updated by this PR can be viewed at: * https://pr47019-757d0c8.ngbuilds.io/guide/contributors-guide-overview

File: aio/content/guide/docs-lint-errors.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+When necessary, you can apply these exceptions to your content.
+
+1.  **General exception**
+
+    A *general exception* allows you to exclude the specified text from all lint testing.
+
+    To apply a general exception, surround the text that you do not want the linter to test with the HTML
+    `comment` elements shown in this example.
+
+    ```markdown
```
Comment (gkalpak) [inline aio/content/guide/docs-lint-errors.md:196]: Our code-highlighter does not recognize `markdown` as a language, so these snippets end up be styled incorrectly.
The next best thing for now is to mark them as `html` (which is the closest to Markdown supported language).

(This applies here and in the rest of the `markdown` snippets in this guide.)

**Review by gkalpak — CHANGES_REQUESTED** [review]
One comment. Otherwise lgtm :+1:

File: aio/content/guide/docs-lint-errors.md (hunk 1 above)
Comment (bob-watson) [reply aio/content/guide/docs-lint-errors.md]: Thanks! Updated in the latest.

**Review by gkalpak — APPROVED** [review]

**Review by MarkTechson — APPROVED** [review]
LGTM

**Comment by dylhunn** [comment]
This PR was merged into the repository by commit 340fa465ceb166f64a99f06596f11deb4b60240b.

**Pull request closed without merging**